src/message.js

```diff
@@ -0,0 +1,160 @@
+import EventObserver from './events';
+import { WEBRTC_DATACHANNEL_CHUNK_SIZE } from './_constants';
+
+export default class Message {
```

Comment:
To avoid conflicting with PySyft Messages, could we maybe opt to name this something else? I like `DataChannelMessage`.

Reply:
Done